android/app/src/main/kotlin/io/lantern/apps/AppsDataProvider.kt

```diff
@@ -11,7 +16,8 @@ class AppsDataProvider(
     private val applicationFilterPredicate: (ApplicationInfo) -> Boolean = { appInfo ->
         hasInternetPermission(appInfo.packageName) &&
                 isLaunchable(appInfo.packageName) &&
-                !isSelfApplication(appInfo.packageName)
+                !isSelfApplication(appInfo.packageName) &&
+                !isSystemApp(appInfo.packageName)
```

Comment:
It looks like the filtering logic here is incorrectly excluding certain apps from the split tunneling list now. For instance, I'm no longer seeing apps that come pre-installed on devices (like Chrome, Gmail, etc).